lib/iris/plot.py

```diff
@@ -481,6 +482,79 @@ def _get_plot_objects(args):
     return u_object, v_object, u, v, args
 
 
+def _get_geodesic_params(globe):
```

Comment:
This should definitely not be in iris. It falls well and truly as a useful method for cartopy's Globe objects. @djkirkham happy to assist?